### PR TITLE
adjust some mob pools that had 0 combat delay

### LINKED ISSUE #618: demon befouler on xarcabard [S] kills you on 1 attack
**_I have:_**

- [x] searched existing issues (http://project-topaz.com/issues/) to see if the issue has already been opened
- [x] checked the commit log to see if the issue has been resolved since my server was last updated

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 

the demon hits as if it has a permanent double attack, that hits like 8 times in a row >.< doubt this is how it is on retail >.<

### PR DESCRIPTION
**Pull request by wrenffxi**
**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

I did a search for mob pools with 0 cmbDelay that were attached to mobs with coordinates.  For each, I infilled some fields from wiki and/or similar pools.  Changes include:

| Mob  | Wiki | Changes  |
| ------------- | ------------- | ------------- |
| Bravo  | [Link](https://ffxiclopedia.fandom.com/wiki/Bravo)  | made SPAWNTYPE_SCRIPTED |
| Ab_xzomit | [Link](https://ffxiclopedia.fandom.com/wiki/Ab'xzomit) | infilled cmbSkill and cmbDelay |
| Aestutaur | [Link](https://ffxiclopedia.fandom.com/wiki/Aestutaur) | infilled cmbSkill and cmbDelay, made linking |
| Akrab | [Link](https://ffxiclopedia.fandom.com/wiki/Akrab) | infilled cmbSkill and cmbDelay |
| Arid_Limule | [Link](https://ffxiclopedia.fandom.com/wiki/Arid_Limule) | infilled cmbSkill and cmbDelay, made BLM, assigned spell list |
| Berserker_Demon | [Link](https://ffxiclopedia.fandom.com/wiki/Berserker_Demon) | infilled cmbSkill and cmbDelay |
| Demon_Befouler | [Link](https://ffxiclopedia.fandom.com/wiki/Demon_Befouler) | infilled cmbSkill and cmbDelay, made DRK, assigned spell list [Fixes #618] |
| Glacial_Imp | [Link](https://ffxiclopedia.fandom.com/wiki/Glacial_Imp) | infilled cmbSkill and cmbDelay, made BLM, made true-aggro and linking, assigned spell list |
| Madthrasher_Zradbodd | [Link](https://ffxiclopedia.fandom.com/wiki/Madthrasher_Zradbodd) | infilled cmbSkill and cmbDelay, made true-aggro linking NM, assigned immunities |
| Snow_Wight | [Link](https://ffxiclopedia.fandom.com/wiki/Snow_Wight) | infilled cmbSkill and cmbDelay, made aggro. Snow Wights can be WAR or BLM. This pool is for WAR. We will need to determine which Snow Wights are BLM from retail, and create a new group and pool for them in another PR later. |
| Spitting_Spider | [Link](https://ffxiclopedia.fandom.com/wiki/Spitting_Spider) | infilled cmbSkill and cmbDelay |

### CODE REVIEW

**Review by ibm2431 — APPROVED** [review]

**Comment by TeoTwawki** [comment]
Surprised this didn’t happen sooner, even if _Million Fists_  is hilarious (when its not me).

File: sql/mob_pools.sql [hunk 1]
```diff
@@ -115,7 +115,7 @@ INSERT INTO `mob_pools` VALUES (54,'Aerns_Elemental','Aerns_Elemental',100,0x000
 INSERT INTO `mob_pools` VALUES (55,'Aerns_Euvhi','Aerns_Euvhi',109,0x00000E0200000000000000000000000000000000,1,1,11,240,100,0,0,0,1,8,0,0,0,1155,5,0,0,1,0,109);
 INSERT INTO `mob_pools` VALUES (56,'Aerns_Wynav','Aerns_Wynav',193,0x0000120200000000000000000000000000000000,1,1,7,200,100,0,0,0,1,8,1,0,0,1155,0,1,0,1,0,714);
 INSERT INTO `mob_pools` VALUES (57,'Aerns_Xzomit','Aerns_Xzomit',269,0x0000100200000000000000000000000000000000,1,1,7,200,100,0,0,0,1,0,0,0,0,1153,0,0,0,1,0,269);
-INSERT INTO `mob_pools` VALUES (58,'Aestutaur','Aestutaur',240,0x00004F0500000000000000000000000000000000,1,1,0,0,100,0,1,1,0,0,0,0,0,0,0,0,0,1,0,240);
+INSERT INTO `mob_pools` VALUES (58,'Aestutaur','Aestutaur',240,0x00004F0500000000000000000000000000000000,1,1,7,240,100,0,1,1,1,0,0,0,0,0,0,0,0,1,0,240);
```
Comment (59blargedy) [inline sql/mob_pools.sql:118]: should we make him a monk? not sure wiki is always right, but that is indicated. I'm new to reviewing so if there are nuances to mob_pools jobs then just let me know!

File: sql/mob_pools.sql [hunk 2]
```diff
@@ -3748,7 +3748,7 @@ INSERT INTO `mob_pools` VALUES (3687,'Snow_Devil','Snow_Devil',232,0x00001601000
 INSERT INTO `mob_pools` VALUES (3688,'Snow_Gigas','Snow_Gigas',126,0x0000C50200000000000000000000000000000000,2,2,1,480,100,0,1,0,1,0,0,0,4143,131,0,0,0,1,0,126);
 INSERT INTO `mob_pools` VALUES (3689,'Snow_Lizard','Snow_Lizard',97,0x0000490100000000000000000000000000000000,1,1,7,240,100,0,0,0,1,0,0,0,239,131,0,0,0,1,0,97);
 INSERT INTO `mob_pools` VALUES (3690,'Snow_Maiden','Snow_Maiden',232,0x0000160100000000000000000000000000000000,1,1,7,240,100,0,1,0,0,2,0,0,0,645,0,0,0,1,0,232);
-INSERT INTO `mob_pools` VALUES (3691,'Snow_Wight','Snow_Wight',227,0x00003C0200000000000000000000000000000000,1,1,0,0,100,0,0,0,0,0,0,0,290,131,0,0,0,1,0,227);
+INSERT INTO `mob_pools` VALUES (3691,'Snow_Wight_war','Snow_Wight',227,0x00003C0200000000000000000000000000000000,1,1,5,240,100,0,1,0,0,0,0,0,290,131,0,0,0,1,0,227);
```
Comment (59blargedy) [inline sql/mob_pools.sql:3751]: should its weapon be 7 for scythe instead? not sure if visual representation always equates to actual weapon though.

Comment (TeoTwawki) [reply sql/mob_pools.sql:3751]: it doesn't. mobs don't have equipment, its just part of their model. He'd do a damage type (slashing, piercing, blunt, h2h) in retail but wouldn't have a "scythe" or  any skill with any weapon type. The attack and accuracy get set in mob_family system and the weapon damage rating and delay are formed from the [mobs level](https://github.com/project-topaz/topaz/blob/5991d15a9a17549ca7a076b7a4a42ee3a26be262/src/map/utils/mobutils.cpp#L72)+dmgMulti and delay fields here in mob_pools, completely ignoring its weapon type field (which afaik only determines if it double swings or not, and possibly wich dmg type to give it by just assuming all swords are slashing etc.)

**Comment by zircon-tpl** [comment]
Merges by staff into `release` have been disabled for the duration of the Return Home to Vana'diel campaign.

Due to the timing of this submission, I will merge this Pull Request into `release` once it has approval from a second staff member.

**Review by 59blargedy — APPROVED** [review]

**Pull request merged**